src/App.jsx

```diff
@@ -14,70 +15,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {!selectedGood
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+        : (
+          <h1
+            className="title is-flex is-align-items-center"
+          >
+            {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => {
+                setGoods('');
+              }}
```

Comment:
```suggestion
              onClick={reset}
```
Consider moving this logic from jsx